sc2reader/resources.py

```diff
@@ -1096,9 +1097,10 @@ def __init__(self, summary_file, filename=None, lang="enUS", **options):
         self.end_time = datetime.utcfromtimestamp(self.parts[0][8])
         self.game_speed = LOBBY_PROPERTIES[0xBB8][1][self.parts[0][0][1].decode("utf8")]
         self.game_length = utils.Length(seconds=self.parts[0][7])
+        expansion = self.expansion if self.expansion else 'LotV'
         self.real_length = utils.Length(
             seconds=int(
```

Comment:
It may confuse readers when code is wrapped just to make room for a comment.
```suggestion
             # Use @brean's logic to fix bot replays
            seconds=int(
```